HacktoberfestProject.Web/Views/Home/Add.cshtml

```diff
@@ -8,50 +8,59 @@
 
 <div class="container">
 	<form asp-controller="Home" asp-action="Add" method="post">
-		<div class="form-group">
-			<label asp-for="PrUrl"></label>
-			<input asp-for="PrUrl" class="form-control" oninput="processURL()" />
-			<span asp-validation-for="PrUrl" class="text-danger"></span>
-		</div>
 		<div class="form-group">
 			<label asp-for="Owner"></label>
-			<input asp-for="Owner" class="form-control" />
+			<input asp-for="Owner" 
+			       class="form-control"
+			       placeholder="Start typing the Github owner name..."
+			       autocomplete="off"/>
 			<span asp-validation-for="Owner" class="text-danger"></span>
 		</div>
+		<div class="form-group">
```

Comment:
Is there a reason for moving the Owner input before the PR Url one? Since we still have `processUrl()` available, we might want to keep that as the first option since it populates all the fields. If a user wants to enter the information manually, they can go to the rest of the fields.

Reply:
It sounded like owner was going to drive the population of other inputs. That’s why it was moved. I don’t think we will have the other JavaScript once all the other drop downs are populated.

Reply:
Ok you changed my mind. At least until this is fully implemented we should keep the previous order and logic too.